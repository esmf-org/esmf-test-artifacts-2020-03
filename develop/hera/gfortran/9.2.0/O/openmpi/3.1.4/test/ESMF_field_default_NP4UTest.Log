build time -- 2021-11-29 19:11:28
20211129 192342.958 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.958 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211129 192342.958 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211129 192342.958 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211129 192342.958 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211129 192342.958 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.958 INFO             PET0 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_03
20211129 192342.958 INFO             PET0 ESMF library build date/time: "Nov 29 2021" "19:10:23"
20211129 192342.958 INFO             PET0 ESMF library build location : /scratch1/NCEPDEV/stmp2/role.esmfmaint/gfortran_9.2.0_openmpi_O_develop
20211129 192342.958 INFO             PET0 ESMF_COMM                   : openmpi
20211129 192342.958 INFO             PET0 ESMF_MOAB                   : enabled
20211129 192342.958 INFO             PET0 ESMF_LAPACK                 : enabled
20211129 192342.958 INFO             PET0 ESMF_NETCDF                 : enabled
20211129 192342.958 INFO             PET0 ESMF_PNETCDF                : disabled
20211129 192342.958 INFO             PET0 ESMF_PIO                    : enabled
20211129 192342.958 INFO             PET0 ESMF_YAMLCPP                : enabled
20211129 192342.959 INFO             PET0  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
20211129 192342.959 INFO             PET0  NUMBER_OF_PROCESSORS           4
20211129 192342.994 INFO             PET0  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
20211129 192342.994 INFO             PET0 Finalizing ESMF
20211129 192342.958 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.958 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211129 192342.958 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211129 192342.958 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211129 192342.958 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211129 192342.958 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.958 INFO             PET1 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_03
20211129 192342.958 INFO             PET1 ESMF library build date/time: "Nov 29 2021" "19:10:23"
20211129 192342.958 INFO             PET1 ESMF library build location : /scratch1/NCEPDEV/stmp2/role.esmfmaint/gfortran_9.2.0_openmpi_O_develop
20211129 192342.958 INFO             PET1 ESMF_COMM                   : openmpi
20211129 192342.958 INFO             PET1 ESMF_MOAB                   : enabled
20211129 192342.958 INFO             PET1 ESMF_LAPACK                 : enabled
20211129 192342.958 INFO             PET1 ESMF_NETCDF                 : enabled
20211129 192342.958 INFO             PET1 ESMF_PNETCDF                : disabled
20211129 192342.958 INFO             PET1 ESMF_PIO                    : enabled
20211129 192342.958 INFO             PET1 ESMF_YAMLCPP                : enabled
20211129 192342.959 INFO             PET1  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
20211129 192342.959 INFO             PET1  NUMBER_OF_PROCESSORS           4
20211129 192342.994 INFO             PET1  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
20211129 192342.994 INFO             PET1 Finalizing ESMF
20211129 192342.958 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.958 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211129 192342.958 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211129 192342.958 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211129 192342.958 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211129 192342.958 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.958 INFO             PET2 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_03
20211129 192342.958 INFO             PET2 ESMF library build date/time: "Nov 29 2021" "19:10:23"
20211129 192342.958 INFO             PET2 ESMF library build location : /scratch1/NCEPDEV/stmp2/role.esmfmaint/gfortran_9.2.0_openmpi_O_develop
20211129 192342.958 INFO             PET2 ESMF_COMM                   : openmpi
20211129 192342.959 INFO             PET2 ESMF_MOAB                   : enabled
20211129 192342.959 INFO             PET2 ESMF_LAPACK                 : enabled
20211129 192342.959 INFO             PET2 ESMF_NETCDF                 : enabled
20211129 192342.959 INFO             PET2 ESMF_PNETCDF                : disabled
20211129 192342.959 INFO             PET2 ESMF_PIO                    : enabled
20211129 192342.959 INFO             PET2 ESMF_YAMLCPP                : enabled
20211129 192342.959 INFO             PET2  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
20211129 192342.959 INFO             PET2  NUMBER_OF_PROCESSORS           4
20211129 192342.994 INFO             PET2  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
20211129 192342.994 INFO             PET2 Finalizing ESMF
20211129 192342.957 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.957 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20211129 192342.957 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20211129 192342.957 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20211129 192342.957 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20211129 192342.957 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20211129 192342.958 INFO             PET3 Running with ESMF Version   : ESMF_8_3_0_beta_snapshot_03
20211129 192342.958 INFO             PET3 ESMF library build date/time: "Nov 29 2021" "19:10:23"
20211129 192342.958 INFO             PET3 ESMF library build location : /scratch1/NCEPDEV/stmp2/role.esmfmaint/gfortran_9.2.0_openmpi_O_develop
20211129 192342.958 INFO             PET3 ESMF_COMM                   : openmpi
20211129 192342.958 INFO             PET3 ESMF_MOAB                   : enabled
20211129 192342.958 INFO             PET3 ESMF_LAPACK                 : enabled
20211129 192342.958 INFO             PET3 ESMF_NETCDF                 : enabled
20211129 192342.958 INFO             PET3 ESMF_PNETCDF                : disabled
20211129 192342.958 INFO             PET3 ESMF_PIO                    : enabled
20211129 192342.958 INFO             PET3 ESMF_YAMLCPP                : enabled
20211129 192342.959 INFO             PET3  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
20211129 192342.959 INFO             PET3  NUMBER_OF_PROCESSORS           4
20211129 192342.994 INFO             PET3  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
20211129 192342.994 INFO             PET3 Finalizing ESMF
