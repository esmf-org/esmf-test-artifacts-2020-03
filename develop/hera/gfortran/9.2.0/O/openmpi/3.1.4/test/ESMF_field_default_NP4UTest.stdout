build time -- 2022-02-20 05:55:56
[1645337190.692101] [h10c36:209541:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1645337190.697178] [h10c36:209541:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1645337190.692154] [h10c36:209539:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1645337190.697250] [h10c36:209539:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1645337190.698736] [h10c36:209540:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1645337190.703699] [h10c36:209540:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1645337190.698752] [h10c36:209538:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
[1645337190.703747] [h10c36:209538:0]         mxm.c:196  MXM  WARN  The 'ulimit -s' on the system is set to 'unlimited'. This may have negative performance implications. Please set the stack size to the default value (10240) 
  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
  NUMBER_OF_PROCESSORS           4
  Beginning Test, file ESMF_TestHarnessUTest.F90, line 74
  NUMBER_OF_PROCESSORS           4
  NUMBER_OF_PROCESSORS           4
command line arg count =    4
command line arg [ 1] =    5:-case
command line arg [ 2] =   21:field_default_test.rc
command line arg [ 3] =    4:-xml
command line arg [ 4] =   17:field_default.xml
argindex (-path) =   -1
argindex (-case) =    1
argindex (-xml) =    3
Path Length =     1
Path = .
Config File = field_default_test.rc
Run Flag =  1
XML Flag =  1
XML File = field_default.xml
   0Path = .
   0Config File = field_default_test.rc
   0Run Flag =  1
   0XML Flag =  1
   0XML File = field_default.xml
   2Path = .
   2Config File = field_default_test.rc
   1Path = .
   2Run Flag =  1
   2XML Flag =  1
   3Path = .
   3Config File = field_default_test.rc
   1Config File = field_default_test.rc
   1Run Flag =  1
   1XML Flag =  1
   2XML File = field_default.xml
   3Run Flag =  1
   3XML Flag =  1
   3XML File = field_default.xml
   1XML File = field_default.xml
 running nonexhaustive tests
 --------------------------------------- 
 Start of test harness test for class FIELD
 --------------------------------------- 
 Opening Dist specifier file  ./field_default_dist_2D.rc
 Opening Dist specifier file  ./field_default_dist_2D.rc
 Opening Grid specifier file  ./field_default_grid_2D.rc
 Opening Grid specifier file  ./field_default_grid_2D.rc
 Opening Dist specifier file  ./field_default_dist_2D.rc
 Opening Dist specifier file  ./field_default_dist_2D.rc
  read grid specification for regridding test
  read grid specification for regridding test
 Opening Grid specifier file  ./field_default_grid_2D.rc
 Opening Grid specifier file  ./field_default_grid_2D.rc
 Problem Descriptor String [B1G1;B2G2] =B=> [B1G1;B2G2]
 ( grid config, distribution config, Grid file, Distribution file)
  read grid specification for regridding test
 (1,1,1,1) string=[B1{1}G1{20};B2{1}G2{20}]=B=>[B1{1}G1{50};B2{1}G2{50}]
                
                
  read grid specification for regridding test
                
                
 -----------------======field regrid test==========-----------------------
field regrid with string [B1G1;B2G2] =B=> [B1G1;B2G2] with entry 1 of file ./field_default_dist_2D.rc and entry 1 of file ./field_default_grid_2D.rc
 Regrid Completed
  PASS: ESMF_TestHarnessUTest.F90 287
PASS: 
 --------------------------------------- 
 End of harness test for class FIELD
 --------------------------------------- 
  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
  Ending Test, file ESMF_TestHarnessUTest.F90, line 299
  PET           3  Test Elapsed Time    51.182000000000002       msec.
  PET           1  Test Elapsed Time    51.933000000000007       msec.
  PET           0  Test Elapsed Time    52.109999999999992       msec.
  PET           2  Test Elapsed Time    52.703999999999972       msec.
